### PR TITLE
chore(monitor): use gwei in fee total metrics

### PR DESCRIPTION
**Pull request by kevinhalliday**
Use gwei in fee metrics, add unit to metric name.

issue: #1962

### CODE REVIEW

File: monitor/xmonitor/indexer/metrics.go [hunk 1]
```diff
@@ -46,10 +50,10 @@ var (
 		Buckets:   prometheus.ExponentialBucketsRange(1, 1e6, 10),
 	}, []string{"stream", "xdapp"})
 
-	feesCollectedTotal = promauto.NewCounterVec(prometheus.CounterOpts{
+	feesGweiTotal = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "monitor",
 		Subsystem: "indexer",
-		Name:      "fees_collected_total",
+		Name:      "fees_gwei_total",
```
Comment (chmllr) [inline monitor/xmonitor/indexer/metrics.go:56]: Nit: did you maybe consider `total_fees_gwei` as it's common to use the unit in a suffix?

Comment (kevinhalliday) [reply monitor/xmonitor/indexer/metrics.go:56]: we have a linter that enforces `_total` as suffix for counter metrics

Comment (corverroos) [reply monitor/xmonitor/indexer/metrics.go:56]: yeah, counters are usually suffixed with `total`

Comment (chmllr) [reply monitor/xmonitor/indexer/metrics.go:56]: Oh I see, those are metrics too 👍

**Review by corverroos — APPROVED** [review]

**Pull request merged**